src/font_description.rs

```diff
@@ -4,14 +4,15 @@
 
 use glib::translate::*;
 use pango_sys;
+use std::ffi::CStr;
 use FontDescription;
 
 impl FontDescription {
-    pub fn set_family_static(&mut self, family: &'static str) {
+    pub fn set_family_static(&mut self, family: &'static CStr) {
```

Comment:
This is rather annoying to use like that. I'd suggest removing the function completely as it doesn't bring much advantage.

@GuillaumeGomez ?

Reply:
I guess so...

Reply:
It's impossible to make it as ergonomic as other methods. I have no problem with it being removed.

Reply:
Let's get rid of it then, if someone wants to call it they can still use the FFI functions.

Reply:
@yvt do you want to update the PR?